components/ui/CompactFeature.vue

```diff
@@ -20,31 +20,45 @@ import { Component, Prop } from 'vue-property-decorator'
 
 @Component
 export default class extends Vue {
-  @Prop(String) icon: string | undefined
-  @Prop(String) title: string | undefined
-  @Prop(String) description: string | undefined
+  @Prop(String) icon!: string
+  @Prop(String) title!: string
+  @Prop(String) description!: string
 }
 </script>
 
 <style lang="scss" scoped>
 @import '~carbon-components/scss/globals/scss/typography';
 
+$feature-icon-size: 2.25rem;
+
 .compact-feature {
-  $feature-icon-size: 3.75rem;
+  background-color: $purple-70;
+  padding: $spacing-05;
+  padding-bottom: $spacing-09;
+
+  @include mq($until: large) {
+    padding-bottom: $spacing-06;
+  }
 
   &__icon {
-    position: relative;
-    width: $feature-icon-size;
     height: $feature-icon-size;
-    margin-bottom: $spacing-05;
+    margin-bottom: $spacing-06;
   }
 
   &__title {
-    @include productive-heading-04_5();
+    @include type-style('productive-heading-02');
+    color: $white;
+    margin-bottom: $spacing-04;
+    min-height: 2.5rem; // for consistent alignment
```

Comment:
Review the alignment, please:
<img width="1052" alt="Screen Shot 2020-11-05 at 09 42 44" src="https://user-images.githubusercontent.com/17231966/98218656-4d500f00-1f4c-11eb-977e-803b9a655813.png">

Reply:
This should be fixed